sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateError.cs

```diff
@@ -8,12 +8,19 @@ namespace Azure.Security.KeyVault.Certificates
     /// <summary>
     /// An error encountered during the processing of a <see cref="CertificateOperation"/>
     /// </summary>
-    public class Error : IJsonDeserializable
+    public class CertificateError : IJsonDeserializable
```

Comment:
Possibly CertificateOperationError? I believe this only comes back as part of a CertificateOperationProperties

Reply:
That would be a change different from what we decided in the v-team before. We'll have to drive that across languages.

Reply:
Actually, given what we discussed earlier with @pakrym, shouldn't this just be captured in the updated `RequestFailedException`? Given the use case, it may not throw but could still be attached, much like in the TPL.

Reply:
I think copying info from error to RequestFailedException would be great